routes/index.js

```diff
@@ -1115,6 +1115,61 @@ router.delete('/api/v1/floorExpense/:id', (req, res, next) => {
   });
 });
 
+/*------------------------ Function Endpoints---------------------------*/
+/* In these API endpoints, they are all labeled as GETs even though they are
+   defined as router.post requests. This is because these functions often 
+   need arguments, and you need to create POST or PUT requests in order to
+   include a body to contain these arguments.
+*/
+
+/* GET attendance of floor given a quarter */
+router.post('/api/v1/floorAttendance', (req, res, next) => {
```

Comment:
Just update the Get /api/v1/members request to include the members' floor and meet_attend. It can be sorted/filtered in js